samples/samples/snippets.py

```diff
@@ -1703,7 +1703,10 @@ def query_data_with_query_options(instance_id, database_id):
     with database.snapshot() as snapshot:
         results = snapshot.execute_sql(
             "SELECT VenueId, VenueName, LastUpdateTime FROM Venues",
-            query_options={"optimizer_version": "1"},
+            query_options={
+                "optimizer_version": "1",
+                "optimizer_statistics_package": "auto_20191128_14_47_22UTC"
```

Comment:
In the samples in other languages, this is swapped around. It's "latest" here and "auto_20191128_14_47_22UTC" in the "spanner_create_client_with_query_options" sample.

Reply:
Whoops! :sweat_smile: